tyr-core/src/test/java/org/jboss/tyr/check/CommitsQuantityCheckTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019-2021 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.tyr.check;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.jboss.tyr.InvalidPayloadException;
+import org.jboss.tyr.TestUtils;
+import org.jboss.tyr.model.yaml.CommitsQuantity;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+
+@QuarkusTest
+public class CommitsQuantityCheckTest {
+
+    @Inject
+    CommitsQuantityCheck commitsQuantityCheck;
+
+    private CommitsQuantity commitsQuantity;
+
+    @BeforeEach
+    public void before() {
+        commitsQuantity = new CommitsQuantity();
+    }
+
+    @Test
+    public void testNullCommitParameter() {
+        Assertions.assertThrows(IllegalArgumentException.class, () -> commitsQuantityCheck.quantityCheck(null));
+    }
+
+    @Test
+    public void testNullCommitsQuantityParameter() {
+        commitsQuantity.setQuantity(null);
+        Assertions.assertThrows(IllegalArgumentException.class, () -> commitsQuantityCheck.quantityCheck(commitsQuantity));
+    }
+
+    @Test
+    public void testNullCommitsMessageParameter() throws InvalidPayloadException {
+        commitsQuantity.setMessage(null);
+        commitsQuantity.setQuantity("5");
+        commitsQuantityCheck.quantityCheck(commitsQuantity);
+
+        Assertions.assertEquals(CommitsQuantityCheck.DEFAULT_MESSAGE, commitsQuantityCheck.check(TestUtils.TEST_PAYLOAD));
+    }
+
+    @Test
+    public void testCheckValueRegexMatchPayloadMatch() throws InvalidPayloadException {
```

Comment:
you shouldn't mentioned tested internals in the test names, what if we change valueRegex field in future?